Assets/HoloToolkit/UX/Scripts/Collections/ObjectCollection.cs

```diff
@@ -62,6 +62,13 @@ public class ObjectCollection : MonoBehaviour
         [Tooltip("Radius for the sphere or cylinder")]
         public float Radius = 2f;
 
+        /// <summary>
+        /// This is the radial range for creating a radial fan layout.
+        /// </summary>
+        [Range(5f, 360f)]
+        [Tooltip("Radial range for radial layout")]
+        public float RadialRange = 180f;
```

Comment:
should be private and serialized, if access outside this class is required, then add a public property

Reply:
Same thing for the rest of the fields below.